.github/workflows/run_all_frameworks.yml

```diff
@@ -35,6 +39,8 @@ jobs:
           echo Common files: $common_files
           echo Changed files: $changed_files
 
+          shopt -u globstar
```

Comment:
not sure you need to turn it off, the enabling should apply only to current shell